### PR TITLE
docs: update some gitignores for new languages

### PR DESCRIPTION
**Pull request by phillc54**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,12 +26,15 @@ src/modules.order
 /configs/*/emc.nml
 !/configs/common/emc.nml
 /docs/src/source-highlight/local
+docs/src/*/gui/gmoccapy_release_notes.txt
```
Comment (smoe) [inline .gitignore:29]: Why?

File: .gitignore [hunk 2]
```diff
@@ -26,12 +26,15 @@ src/modules.order
...
 lib/python/gremlin.py
 lib/python/qt5_graphics.py
 lib/python/touchy
 configs/sim/gmoccapy/*.pref
 configs/sim/gmoccapy/gmoccapy_plasma/*.pref
 configs/sim/gmoccapy/gmoccapy-JA/*.pref
+configs/**/*.tbl.bak
```
Comment (smoe) [inline .gitignore:36]: Maybe even *.bak would make sense as a global rule?

File: .gitignore (hunk 1 above)
Comment (phillc54) [reply .gitignore:29]: They are pre existing, I just moved them near more aligned rules as they looked quite silly following the commented line `# Ignore VSCode settings`

File: .gitignore (hunk 2 above)
Comment (petterreinholdtsen) [reply .gitignore:36]: Perhaps, but that can wait for another patch.

**Pull request merged**